### ISSUE TITLE
"IOError: [Errno 29] Illegal seek" when LOG_FILE = /dev/stdout

### ISSUE THREAD

**Issue by AlexClineBB** [Netflix/security_monkey#246]
I've installed Security Monkey within a docker container. To ensure that I'm able to collect the logs, I've configured logging with:

    LOG_LEVEL = "DEBUG"
    LOG_FILE = "/dev/stdout"

When the scheduler is running, it throws the following error for every event it tries to write to the log (the python file triggering the bug is different, but the error is the same):

```
Traceback (most recent call last):
  File "/usr/lib/python2.7/logging/handlers.py", line 76, in emit
    if self.shouldRollover(record):
  File "/usr/lib/python2.7/logging/handlers.py", line 157, in shouldRollover
    self.stream.seek(0, 2)  #due to non-posix-compliant Windows feature
IOError: [Errno 29] Illegal seek
Logged from file auditor.py, line 273
```

**Comment by monkeysecurity** [comment]
Yeah I've seen this before as well when playing with dockerizing security_monkey.  I'm assuming it has something to do with the RotatingFileHandler or StreamHandler, but never really dug into it:

https://github.com/Netflix/security_monkey/blob/master/security_monkey/__init__.py#L39

**Comment by AlexClineBB** [comment]
You're right.

To test, I tried changing the `maxBytes` and `backupCount` parameters to zero (0). This caused the RotatingFileHandler to never try to rotate the log and didn't issue the illegal seek.

In my case, rather than changing the source, I was able to work around the error by specifying `LOG_FILE = "/dev/null"`. The docker container running the scheduler was able to capture the scheduler's output without having to direct it to /dev/stdout. It's only missing the scheduler-generated timestamps which I don't need currently.

I'll close this ticket since there are workarounds available.
